api/src/pcapi/local_providers/titelive_things/titelive_things.py

```diff
@@ -413,9 +413,9 @@ def postTreatment(self) -> None:
             try:
                 offers_api.approves_provider_product_and_rejected_offers(ean)
             except ProductNotFound as exception:
-                logger.error("Imported product with ean %s not found", extra={"ean": ean, "exc": str(exception)})
+                logger.error("Imported product with ean %s not found", ean, extra={"exc": str(exception)})
```

Comment:
On préfère en général mettre ce qui est variable dans les `extra`, car ça facilite la recherche dans les logs sur GCP.
Je propose donc de corriger plutôt ainsi : 
```suggestion
                logger.error("Imported product with ean not found", ean, extra={"ean": ean, "exc": str(exception)})
```
Cela permettra de faire une requête sur un EAN particulier dans GCP : 
`jsonPayload.extra.ean="1234567890123"`
ou pour tous les trouver, avec un message fixe : 
`jsonPayload.message="Imported product with ean not found"`

Pareil pour le suivant deux lignes plus bas.

Reply:
Changed